etl/steps/data/garden/urbanization/2024-01-26/ghsl_degree_of_urbanisation.meta.yml

```diff
@@ -0,0 +1,206 @@
+definitions:
+  common:
+    presentation:
+      topic_tags:
+        - Urbanization
+    processing_level: minor
+    description_key:
+    - |-
+      **The Degree of Urbanisation (DEGURBA)** is a  method designed for capturing the urban-rural continuum and facilitating international comparisons. Developed by six international organizations and endorsed by the UN Statistical Commission, it employs a two-level classification system.
+
+      The first level categorizes territories into three classes: 1) cities, 2) towns and suburbs, and 3) rural areas. This classification helps distinguish urban areas (cities plus towns and suburbs) from rural areas, emphasizing the distinct differences between towns, semi-dense areas, cities, and rural regions.
+
+      The second level adds granularity by further dividing towns and semi-dense areas into towns and suburban or peri-urban areas, and splitting rural areas into villages, dispersed rural areas, and mostly uninhabited areas.
+
+      The classification process involves two steps. Initially, all 1 km² grid cells are categorized into one of three types: urban centers (contiguous grid cells with a density of at least 1500 inhabitants per km² and a total population of 50,000 or more, defining a city), urban clusters (contiguous grid cells with a density of at least 300 inhabitants per km² and a total population of 5,000 or more, defining a town and suburb), and rural grid cells (defining rural areas). These grid cell types are then used to classify smaller administrative or statistical spatial units.
+
+      This method is used with a residential population grid derived from household point locations, typically from geo-coded censuses or registers. Over 30 countries have published such data, and others are collecting it. If point data is unavailable, a disaggregation grid is used, with accuracy dependent on the spatial unit size.
```

Comment:
Only over 30 countries have published data, but the dataset has data for all countries. Is it that the DEGRUBA model is trained on data from these >30 countries and applied to satellite imagery from all countries to create the dataset?

Reply:
good point - I am not 100% sure if that's what they mean by disaggregation grid. I think if they don't have such high-resolution data for a country they then use the population estimates at a lower spatial resolution to divide the country grid into smaller sections and estimate where people might live within that area. I can't find any more detail on this in the paper though.. will try to get in touch with them regarding this!